src/main/java/eu/europeana/metis/sandbox/domain/RecordError.java

```diff
@@ -21,14 +22,15 @@
    */
   public RecordError(RecordProcessingException exception) {
     requireNonNull(exception, "Exception must not be null");
-    this.message = exception.getReportMessage();
+    this.message = Optional.ofNullable(exception.getReportMessage())
+        .orElse("No message. Report to service provider");
```

Comment:
Avoiding error message to be null, after dealing with some tests I realize it was a mistake not to check for this